multi-node/aws/pkg/config/templates/cloud-config-controller

```diff
@@ -83,6 +100,19 @@ write_files:
           "http://127.0.0.1:8080/api/v1/namespaces/kube-system/services"
       done
 
+  - path: /opt/bin/decrypt-tls-assets
+    owner: root:root
+    permissions: 0700
+    content: |
+      #!/bin/bash -e
+
+      for encKey in $(find /etc/kubernetes/ssl/*.pem);do
+        tmpPath="/tmp/$(basename $encKey).tmp"
+        docker run --rm -v /etc/kubernetes/ssl:/etc/kubernetes/ssl --rm quay.io/coreos/awscli aws --region {{.Region}} kms decrypt --ciphertext-blob fileb://$encKey --output text --query Plaintext | base64 --decode > $tmpPath
```

Comment:
It looks like you're overwriting the assets in-place. Is this safe to run multiple times on the same assets (e.g. after reboot)?